src/modules/drops/screens/Drops/components/StoriesSlider/StoriesSlider.tsx

```diff
@@ -51,11 +51,41 @@ export const StoriesSlider = () => {
     requestKey: DROPS_LIVE_KEY,
   });
 
+  const {
+    data: dataComing,
+    loading: loadingComing,
+  } = useQuery<IGetDrops | null>({
+    type: getDrops.toString(),
+    requestKey: DROPS_COMING_KEY,
+  });
+
+  const { data: dataPrev, loading: loadingPrev } = useQuery<IGetDrops | null>({
+    type: getDrops.toString(),
+    requestKey: DROPS_PREV_KEY,
+  });
+
   const liveDrops = data?.items || [];
+  const comingDrops = dataComing?.items || [];
+  const prevDrops = dataPrev?.items || [];
 
-  const renderedItems = liveDrops.map(item => {
-    const chips = <StoriesChip label="live" isLive />;
+  const showDrop =
```

Comment:
const showDrop = data?.items ?? dataComing?.items ?? dataPrev?.items

delete  liveDrops and comingDrops and prevDrops